src/DeploymentManager/DeploymentManager/help/Remove-AzDeploymentManagerArtifactSource.md

```diff
@@ -0,0 +1,215 @@
+Module Name: Az.DeploymentManager
+online version: https://docs.microsoft.com/en-us/powershell/module/az.deploymentmanager/remove-azdeploymentmanagerartifactsource
+schema: 2.0.0
+---
+
+# Remove-AzDeploymentManagerArtifactSource
+
+## SYNOPSIS
+Deletes the specified artifact source.
+
+## SYNTAX
+
+### Interactive (Default)
+```
+Remove-AzDeploymentManagerArtifactSource [-ResourceGroupName] <String> [-Name] <String> [-Force] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceId
+```
+Remove-AzDeploymentManagerArtifactSource [-ResourceId] <String> [-Force] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### InputObject
+```
+Remove-AzDeploymentManagerArtifactSource [-ArtifactSource] <PSArtifactSource> [-Force] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Remove-AzDeploymentManagerArtifactSource** cmdlet deletes an artifact source
+Specify the artifact source by its name and resource group name. Alternately, you can provide the ArtifactSource object or the ResourceId.
+
+## EXAMPLES
+
+### Example 1: Delete an artifact source
+### Example 1
+```powershell
+PS C:\> Remove-AzDeploymentManagerArtifactSource -ResourceGroupName "ContosoResourceGroup" -Name "ContosoArtifactSource"
+```
+
+This command deletes an artifact source named ContosoArtifactSource in ContosoResourceGroup.
+
+### Example 2: Delete an artifact source using the resource identifier
+```powershell
+PS C:\> Remove-AzDeploymentManagerArtifactSource -ResourceId "/subscriptions/subscriptionId/resourcegroups/ContosoResourceGroup/providers/Microsoft.DeploymentManager/artifactSources/ContosoArtifactSource"
+```
+
+This command deletes an artifact source named ContosoArtifactSource in ContosoResourceGroup.
+
+### Example 3: Delete an artifact source using an object
+```powershell
+PS C:\> Remove-AzDeploymentManagerArtifactSource -ArtifactSource $artifactSourceObject
+```
+
+This command deletes an artifact source whose name and ResourceGroup match the Name and ResourceGroupName properties of the $artifactSourceObject, respectively.
+
+## PARAMETERS
+
+### -ArtifactSource
+The artifact source to be removed.
+
+```yaml
+Type: Microsoft.Azure.Commands.DeploymentManager.Models.PSArtifactSource
+Parameter Sets: InputObject
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Force
```

Comment:
@deveshguha you'll need to update the markdown help files since you've made some changes to the public API of the cmdlets over the last few iterations -- you can do this by using the `Update-MarkdownHelpModule` cmdlet that's called out in the [`help-generation.md`](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module) file